<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLinkDialog</class>
 <widget class="QDialog" name="AddLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_addLinkHeader">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;h2&gt;Add new link&lt;/h2&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_linked">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="table_addLinkScientist">
         <property name="styleSheet">
          <string notr="true">selection-background-color: rgb(0, 85, 255);
selection-color: rgb(255, 255, 255);</string>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Scientist</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_addLinkComputer">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Computer</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_addLinkLink">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_addLinkCancel">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
